cameo/flux_analysis/analysis.py

```diff
@@ -169,7 +169,7 @@ def find_blocked_reactions(model):
 
     """
     with model:
-        for exchange in model.exchanges:
+        for exchange in model.boundary:
```

Comment:
I would remove this entire function if easily possible. It is identical to the cobrapy version (which uses `model.exchanges`).